supertokens_python/querier.py

```diff
@@ -179,6 +236,7 @@ async def send_post_request(
         self,
         path: NormalisedURLPath,
         data: Union[Dict[str, Any], None] = None,
+        user_context: Union[Dict[str, Any], None] = None,
```

Comment:
do not make it none by default, otherwise you may miss on passing it somewhere. This also means that you have to make `data` not None by default, which we should do.

Reply:
Same goes for `user_context` inputs for other functions